server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -128,7 +128,15 @@ class QueryBuilderContext:
     table_name: str
 
     def materialize(self) -> QueryBuilder:
-        return self.builder.from_(self.table_name).select(*self.columns)
+        # In case we have no columns, select everything by default.
```

Comment:
i feel like it could be interessant to add an if check here on the step name too... what do you think ? (no need ? too much ?)

Reply:
We can't, as the step name is not available here anymore